src/BenchmarksServer/Startup.cs

```diff
@@ -211,28 +211,12 @@ private static string CloneAndRestore(string path, ServerJob job)
 
             Debug.Assert(benchmarksDir != null);
 
-            // Modify all global.json to reference source dirs
```

Comment:
Are we :ok: immediately losing the ability to use older branches? E.g. could temporarily make this conditional on the global.json file existing. Remove it a week or three down the road.